web/src/components/molecules/Member/MemberAddModal/index.tsx

```diff
@@ -7,47 +7,42 @@ import Form from "@reearth-cms/components/atoms/Form";
 import Icon from "@reearth-cms/components/atoms/Icon";
 import Input from "@reearth-cms/components/atoms/Input";
 import Modal from "@reearth-cms/components/atoms/Modal";
+import { User } from "@reearth-cms/components/molecules/Member/types";
+import { MemberInput } from "@reearth-cms/components/molecules/Workspace/types";
 import { useT } from "@reearth-cms/i18n";
 
 export interface FormValues {
   name: string;
+  names: User[];
 }
 
 export interface Props {
   open?: boolean;
   handleUserSearch: (nameOrEmail: string) => "" | Promise<any>;
+  handleUserAdd: () => void;
   onClose?: (refetch?: boolean) => void;
-  onSubmit?: (userIds: string[]) => void;
-  searchedUser:
-    | {
-        id: string;
-        name: string;
-        email: string;
-      }
-    | undefined;
-  changeSearchedUser: React.Dispatch<
-    React.SetStateAction<
-      | {
-          id: string;
-          name: string;
-          email: string;
-        }
-      | undefined
-    >
-  >;
+  onSubmit?: (users: MemberInput[]) => void;
+  searchedUser: User | undefined;
+  changeSearchedUser: React.Dispatch<React.SetStateAction<User | undefined>>;
```

Comment:
I think for useState setters you can type it as () => void;

Reply:
Done!
I did this for `changeSearchedUser` 
however for `changeSearchedUserList: React.Dispatch<React.SetStateAction<User[]>>`
I kept it the way it was because of this code:
```js
      changeSearchedUserList((oldList: User[]) =>
        oldList.filter((user: User) => user.id !== userId),
      );
```